tests/unit/mixins/dom-test.js

```diff
@@ -17,6 +18,7 @@ moduleForComponent('ember-lifeline/mixins/dom', {
 
     this.owner = getOwner(this);
     this.owner.register(name, Component.extend(ContextBoundEventListenersMixin, {
+      tagName: 'div',
```

Comment:
That is a good question. Maybe you can answer it better than I. When i dumped the component instance locally it was "null" and the assert checking this.tagName !== '' was passing in cases where null was passed which is the opposite of what we wanted.

Reply:
The default value for `Ember.Component.prototype.tagName` is `null`. If `component.tagName` is `null` when we are rendering, we default to `div` otherwise we use the user provided string.

Reply:
yeah that is the part I was missing. I wasn't familiar with the mechanism used to set the tagName to div and i was expecting to see "div" in cases where it was not explicitly set. So my change and the assertions were based on incorrect assumption about what to expect. Thanks, makes it much less confusing now!